### PR TITLE
Print loaded directories with packages

### PR DESCRIPTION
**Pull request by mtojek**
Fixes: https://github.com/elastic/elastic-package/issues/189

This PR adds log messages showing which directories will be included in the package-registry custom Docker image.

_I'm not sure if this is the expectation, but hopefully it will help understand the logic behind `stack up`._

### CODE REVIEW

File: internal/stack/boot.go [hunk 1]
```diff
@@ -43,6 +43,13 @@ func BootUp(options Options) error {
 		}
 	}
 
+	fmt.Println("Packages from the following directories will be loaded into the package-registry:")
+	fmt.Println("- built-in packages (package-storage:snapshot Docker image)")
+
+	if found {
+		fmt.Printf("- %s (copy of: %s)\n", stackPackagesDir, buildPackagesPath)
```
Comment (ycombinator) [inline internal/stack/boot.go:50]: I think we can just talk about `buildPackagesPath` here. Not sure if we should talk about `stackPackagePath` as it exposes `elastic-package` internals?

**Review by ycombinator — COMMENTED** [review]
Left a minor comment/question.

File: internal/stack/boot.go (hunk 1 above)
Comment (mtojek) [reply internal/stack/boot.go:50]: Maybe you're right, I will leave only the `buildPackagesPath`

Comment (mtojek) [reply internal/stack/boot.go]: Fixed.

**Review by ycombinator — APPROVED** [review]
LGTM.

**Pull request merged**